atlasdb-jepsen-tests/test/jepsen/atlasdb_test.clj

```diff
@@ -1,14 +1,22 @@
 (ns jepsen.atlasdb-test
   (:require [clojure.test :refer :all]
+            [jepsen.nemesis :as nemesis]
             [jepsen.atlasdb.lock :as lock]
+            [jepsen.atlasdb.timelock :as timelock]
             [jepsen.atlasdb.timestamp :as timestamp]
             [jepsen.core :as jepsen]))
 
 ;; Using Clojure's testing framework, we initiate our test runs
 ;; Tests successful iff the value for the key ":valid?" is truthy
 
-(deftest timestamp-test
-   (is (:valid? (:results (jepsen/run! (timestamp/timestamp-test))))))
+(deftest timestamp-test-crash
+  (is (:valid? (:results (jepsen/run! (timestamp/timestamp-test timelock/crash-nemesis))))))
 
-(deftest lock-test
-   (is (:valid? (:results (jepsen/run! (lock/lock-test))))))
+(deftest timestamp-test-partition
+  (is (:valid? (:results (jepsen/run! (timestamp/timestamp-test (nemesis/partition-random-halves)))))))
```

Comment:
why the extra brackets here? `(nemesis/partition-random-halves)` is going to call the `nemesis/partition-random-halves` function, but I think all we want to do is to pass it in (i.e. as a functor/function object). Are you sure this is working?